Yoast/wordpress-seo
# Replace keyword help text by a question mark link

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* In order to position the question mark link,
* It would make it a lot more complex to make the question mark and brackets unbolded. That's why I decided to keep it bold. @hedgefield does agree.

## Test instructions

This PR can be tested by following these steps:

* Link https://github.com/Yoast/yoast-components/pull/757
* Look at the metabox and sidebar and see if they match the screenshots from the issue.

## Quality assurance


Fixes Yoast/wordpress-seo#11235

Requires Yoast/yoast-components#757
